test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -0,0 +1,3665 @@
+          airloop = zone.airLoopHVAC.get
+          assert(!dcv_is_on(zone, airloop))
+
+        when 7
+          # test 7
+          #   - DCV should NOT be in the user model (zone ppl density < 25 ppl/ksqft)
+          #   - DCV should NOT be in the baseline model (air loop oa flow < 3000 cfm || zone ppl density < 100 ppl/ksqft)
+          #   - DCV is implemented in user model
+          # expected result: no DCV in baseline model
+          # test case setting:
+          #   - Kitchen
+          #   - user model air loop oa flow 528 cfm
+          #   - zone ppl density 14.93
+          #   - no user exception
+          #   - DCV implemented in the user model
+          #     - at the airloop level [through modifier]
+          #     - at the zone level, zone oa spec per person 0.003539605824
+          zone = model_baseline.getThermalZoneByName('Kitchen_ZN_1_FLR_1 ZN').get
+          airloop = zone.airLoopHVAC.get
+          assert(!dcv_is_on(zone, airloop))
+
+        when 8
+          # test 8
+          #   - DCV should NOT be in the user model (zone ppl denstiy < 25 ppl/ksqft)
+          #   - DCV should NOT be in the baseline model (air loop oa flow < 3000 cfm || zone ppl density < 100 ppl/ksqft)
+          #   - DCV is NOT implemented in user model
+          # expected result: no DCV in baseline model
+          # test case setting:
+          #   - Kitchen
+          #   - user model air loop oa flow 528 cfm
+          #   - zone ppl density 14.93
+          #   - no user exception
+          #   - DCV not implemented in the user model
+          #     - at the airloop level by default
+          #     - at the zone level [through remove_zone_oa_per_person_spec modifier] (optional)
+          zone = model_baseline.getThermalZoneByName('Kitchen_ZN_1_FLR_1 ZN').get
+          airloop = zone.airLoopHVAC.get
+          assert(!dcv_is_on(zone, airloop))
+        else
+          assert(false, "ERROR! #{tc_id} not a valid test case id for check_dcv")
+        end
+      end
+    end
+  end
+
+  def dcv_is_on(thermal_zone, air_loop_hvac)
+    # check air loop level DCV enabled
+    return false unless air_loop_hvac.airLoopHVACOutdoorAirSystem.is_initialized
+
+    oa_system = air_loop_hvac.airLoopHVACOutdoorAirSystem.get
+    controller_oa = oa_system.getControllerOutdoorAir
+    controller_mv = controller_oa.controllerMechanicalVentilation
+    return false unless controller_mv.demandControlledVentilation == true
+
+    # check zone OA flow per person > 0
+    zone_dcv = false
+    thermal_zone.spaces.each do |space|
+      dsn_oa = space.designSpecificationOutdoorAir
+      next if dsn_oa.empty?
+
+      dsn_oa = dsn_oa.get
+      next if dsn_oa.outdoorAirMethod == 'Maximum'
+
+      if dsn_oa.outdoorAirFlowperPerson > 0
+        # only in this case the thermal zone is considered to be implemented with DCV
+        zone_dcv = true
+      end
+    end
+
+    return zone_dcv
+  end
+
+  def mark_test_case_no(model, arguments)
```

Comment:
what is this method for?

Reply:
@mdahlhausen: It looks like this was added by Jerry Lei, who is on vacation this week. I'm inclined to leave it until we can review with him.

Reply:
Removed.

Reply:
Removing this caused tests to fail, so I've added it back in. It is called based on information in the test case file (test/90_1_prm/data/prototype_list.json).